src/main/java/org/gridsuite/modification/server/entities/equipment/modification/ScalingEntity.java

```diff
@@ -27,11 +38,45 @@
 @Setter
 @MappedSuperclass
 public class ScalingEntity extends ModificationEntity {
+
     @Column(name = "VariationType")
     @Enumerated(EnumType.STRING)
     private VariationType variationType;
 
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    private List<ScalingVariationEntity> variations;
+
     public ScalingEntity(@NotNull ModificationType modificationType) {
         super(modificationType);
     }
+
+    @Override
+    public void update(@NonNull ModificationInfos modificationInfos) {
+        super.update(modificationInfos);
+        assignAttribute((ScalingInfos) modificationInfos);
+    }
+
+    protected void assignAttribute(ScalingInfos scalingInfos) {
```

Comment:
It's better to make it private like others network modifications
This method assign attributes for the local class only
It's better for comprehension